### PR TITLE
Update jsonexample

### PR DESCRIPTION
**Pull request by kevmoo**
- use latest dependencies
- update build.yaml for removed option
- rebuilt generated code

### CODE REVIEW

File: jsonexample/pubspec.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ dependencies:
 dev_dependencies:
   build_runner: ^1.0.0
   built_value_generator: ^6.1.5
-  json_serializable: ^1.0.0
+  json_serializable: ^2.2.1
```
Comment (RedBrogdon) [inline jsonexample/pubspec.yaml]: Whoa, yeah, thanks for catching this. I need to figure out how to hook into pub's "There's a new version you're not getting" warnings.

File: jsonexample/build.yaml [hunk 2]
```diff
@@ -14,13 +14,3 @@ targets:
       json_serializable|json_serializable:
         generate_for:
         - lib/json_serializable/*.dart
-        options:
```
Comment (RedBrogdon) [inline jsonexample/build.yaml]: Did the header option get removed from source_gen/builder_runner?

Comment (kevmoo) [reply jsonexample/build.yaml]: yup

File: jsonexample/pubspec.yaml (hunk 1 above)
Comment (kevmoo) [reply jsonexample/pubspec.yaml]: Yeah, you could look at the output from `pub upgrade` – but flutter hides it TOO well.

**Comment by kevmoo** [comment]
@RedBrogdon – also confirmed I can land PRs here – good to know!

**Pull request merged**